### DISCUSSION TITLE
Error: unrecognised arguments

### DISCUSSION THREAD

**Discussion by Akshaya-Devadiga** [NVIDIA-NeMo/NeMo#5518] [Q&A]
Dear Team,
we are working on pretraining of BERT model with our custom data. 
using bert_pretraining.py code for the same. But while providing with arguments at runtime, it throws an error saying unrecognized arguments

**Comment by blahBlahhhJ** [comment]
In hydra, if you want to provide arguments that are not present in the base config yaml file, you need to append a "+" at the beginning.

e.g. if I want to modify the `library` entry and add an entry called `custom_entry` into `model.tokenizer`, I should do
```
python bert_pretraining.py model.tokenizer.library=megatron +model.tokenizer.custom_entry=42
